<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.669, y2:0.897727, stop:0 rgba(40, 66, 72, 255), stop:1 rgba(79, 123, 127, 255));}</string>
   </property>
   <widget class="QPushButton" name="btn_anmelden">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>260</y>
      <width>361</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: 700 18pt &quot;MS Shell Dig 2&quot;;
background-color: white;
color:black;

</string>
    </property>
    <property name="text">
     <string>Anmelden</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>211</width>
      <height>78</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;MS Shell Dig 2&quot;;color:white;</string>
    </property>
    <property name="text">
     <string>Startseite</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_erstellen">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>400</y>
      <width>361</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: 700 18pt &quot;MS Shell Dig 2&quot;;
background-color: white;
color:black;

</string>
    </property>
    <property name="text">
     <string>Account Erstellen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>170</y>
      <width>61</width>
      <height>78</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dig 2&quot;;color:white;</string>
    </property>
    <property name="text">
     <string>Yurr</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
